js/apps/admin-ui/src/user/UserGroups.tsx

```diff
@@ -237,10 +157,9 @@ export const UserGroups = ({ user }: UserGroupsProps) => {
           }}
           canBrowse={isManager}
           onClose={() => setOpen(false)}
-          onConfirm={(groups) => {
-            addGroups(groups || []);
+          onConfirm={async (groups) => {
+            await addGroups(groups || []);
```

Comment:
You could add a default here instead.
```suggestion
          onConfirm={async (groups = []) => {
            await addGroups(groups);
```